### DISCUSSION TITLE
How do I properly exist neovim once a timer started?

### DISCUSSION THREAD

**Discussion by mainrs** [nvzone/timerly#7] [Q&A] [answered]
Doing a `:q` results in an error:

```
E37: No write since last change
E162: No write since last change for buffer "[Prompt]"
```

I then get a new buffer displayed that contains the text "󰄉  Enter time:". I assume the enter time field from the timerly dialog is still hanging around. `Entering :q!` forces neovim to close then.

**Comment by siduck** [comment, marked as answer]
@mainrs  press "q" on the timer window

**Reply by mainrs** [reply]
Thank you!
